plugins/spy.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: spy
+spec:
+  homepage: https://github.com/huazhihao/kubespy
+  shortDescription: pod debugging tool for kubernetes clusters with docker runtimes
+  version: v0.3.1
+  description: |
```

Comment:
Can you wrap lines at ~80 chars, so that it prints nicely on smaller terminals?

Reply:
Somehow that line wrapping must have slipped through. It greatly improves readability on small terminals, so I recommend doing this.